puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/test_foreman.groovy

```diff
@@ -29,14 +29,40 @@ pipeline {
                                 configureRVM(env.RUBY_VER, env.GEMSET)
                                 databaseFile(gemset(env.GEMSET))
                                 configureDatabase(env.RUBY_VER, env.GEMSET)
-                                withRVM(['npm install'], env.RUBY_VER, env.GEMSET)
                             }
                         }
                         stage("unit-tests-2.5-postgres") {
                             steps {
                                 withRVM(['bundle exec rake jenkins:unit TESTOPTS="-v" --trace'], env.RUBY_VER, env.GEMSET)
                             }
                         }
+                    }
+                    post {
+                        always {
+                            cleanup(env.RUBY_VER, env.GEMSET)
+                            deleteDir()
+                        }
+                    }
+                }
+                stage('ruby-2.5-postgres-integrations') {
+                    agent { label 'fast' }
+                    environment {
+                        RUBY_VER = '2.5'
+                        GEMSET = 'ruby-2.5-postgres'
```

Comment:
Should this be a different `GEMSET` name than in the regular PostgreSQL+Ruby2.5 run?

Reply:
I don't think so, the aim is simply to run the integration tests in
parallel vs serial today.

Reply:
I'm wondering about concurrency issues. What happens if one finishes and cleans up the gemset?

Reply:
I think it should to prevent clashing, or one parallel deleting the gemset before the other is done with it.

Reply:
oh, I was not sure this was actually shared and deleted each time,  I thought it will save some time by not downloading it again.